### PR TITLE
Add support for sending data from webview to native in iOS and Android

### PR DESCRIPTION
**Pull request by OPY-bbt**
* create the local http-server in example/html folder (for example: by [http-server](https://www.npmjs.com/package/http-server))
* modify the messageUrl in main.dart to your http-server url
* run the example project and click the `Open Webview (message demo)`button
* click send to flutter button in webview and you can see the result in the bottom.

### CODE REVIEW

File: android/src/main/java/com/flutter_webview_plugin/WebviewManager.java [hunk 1]
```diff
@@ -495,4 +498,13 @@ void stopLoading(MethodCall call, MethodChannel.Result result){
             webView.stopLoading();
         }
     }
+
+    public class WebAppInterface {
+        @JavascriptInterface
+        public void postMessage(String value){
+            Map<String, String> postMessageMap = new HashMap<>();
+            postMessageMap.put("postMessage", value);
+            FlutterWebviewPlugin.channel.invokeMethod("onPostMessage", postMessageMap);
```
Comment (BobEve) [inline android/src/main/java/com/flutter_webview_plugin/WebviewManager.java]: I have a error:
java.lang.RuntimeException: Methods marked with @UiThread must be executed on the main thread. Current thread: JavaBridge
maybe use 'new Handler(Looper.getMainLooper())'

Comment (OPY-bbt) [reply android/src/main/java/com/flutter_webview_plugin/WebviewManager.java]: done

**Comment by ashbin** [comment]
#71

**Comment by diegovar** [comment]
Can this be merged in?

**Comment by kevinpthorne** [comment]
Please? x)

**Comment by rinlv** [comment]
still waiting 🔢

**Comment by rinlv** [comment]
I solved this problem with dynamic channel name for Android and iOS #523. Hope it's helpful

**Comment by ragaisis** [comment]
This one is is important for us too

**Comment by hungbu** [comment]
Please resolve and merge this request

**Comment by OPY-bbt** [comment]
This is already implemented on the main branch, see example.

File: android/src/main/java/com/flutter_webview_plugin/WebviewManager.java (hunk 1 above)
Comment (vijayawoshe) [reply android/src/main/java/com/flutter_webview_plugin/WebviewManager.java]: How did you solve this error 
?
I have a error:
java.lang.RuntimeException: Methods marked with @UiThread must be executed on the main thread. Current thread: JavaBridge
maybe use 'new Handler(Looper.getMainLooper())'
I am getting the same

Comment (OPY-bbt) [reply android/src/main/java/com/flutter_webview_plugin/WebviewManager.java]: I add mainLooper in [commit](https://github.com/fluttercommunity/flutter_webview_plugin/pull/457/commits/62872ced09f8dce9119f5119086437090f01984e#diff-2965422868f060f130659de350f7c684R510), are you sure on the lastest branch?

Comment (OPY-bbt) [reply android/src/main/java/com/flutter_webview_plugin/WebviewManager.java]: you can look #523.

Comment (OPY-bbt) [reply android/src/main/java/com/flutter_webview_plugin/WebviewManager.java]: #523 is already implemented on the main branch

**Pull request closed without merging**